Random Forest/random_forest/random_forest.py

```diff
@@ -1,24 +1,85 @@
-import pandas as pd
```

Comment:
You use lot of numpy functions in the random forest class but you deleted out numpy import.Please import numpy properly..

Reply:
#64 #11 #44 are good examples of proper implementation by your fellow participants